### PR TITLE
feat(sd): vscode highlighting 2

### PR DESCRIPTION
**Pull request by filipetrevizan**
# MR Title
Provide a brief and descriptive title for the pull request.

## Issue Reference:
Link to the Issue or User Story this PR addresses (Ticket to JIRA if applicable).

## Description of Changes:
- Briefly describe the changes made in this PR.
- Mention any new dependencies or tools added.

## Testing Steps:
- Describe how the changes can be tested. Include any specific commands to run or environments to set up. Mention any specific scenarios that need to be tested.

## Screenshots:
Include screenshots of the new or updated UI/UX, if applicable.

### Reviewer Checklist:
- [] I have added tests that prove my fix is effective or that my feature works.
- [] New and existing unit tests pass locally with my changes.
- [] The code follows the project’s style guidelines.
- [] The changes have been tested and are functioning correctly.
- [] The changes are well-documented.

### Performance Considerations (optional):
- Mention any performance impacts or improvements.
- Any specific areas where reviewers should pay special attention to performance?

### Additional Notes (optional):
Add any other notes or comments that could be helpful for reviewers.
Mention any follow-up tasks or related issues.

### CODE REVIEW

File: sowingdev/app/components/Navigation.tsx [hunk 1]
```diff
@@ -48,7 +48,7 @@ const Navigation: React.FC = () => {
                     </li>
                   </>
                 )}
-                {isAuthenticated && !isLoading && (
+                {!isAuthenticated && !isLoading && (
```
Comment (alejandrochang) [inline sowingdev/app/components/Navigation.tsx]: Remove this

File: sowingdev/middleware.ts [hunk 2]
```diff
@@ -1,4 +1,4 @@
 export { default } from "next-auth/middleware";
 
 // Update for private auth routes
-export const config = { matcher: ['/curriculum']}
```
Comment (alejandrochang) [inline sowingdev/middleware.ts:4]: Revert this, we need this as well

File: sowingdev/app/components/Navigation.tsx (hunk 1 above)
Comment (filipetrevizan) [reply sowingdev/app/components/Navigation.tsx]: Done

File: sowingdev/middleware.ts (hunk 2 above)
Comment (filipetrevizan) [reply sowingdev/middleware.ts:4]: Done

**Pull request merged**